tools/python/boutiques/schema/examples/example1/exampleTool1_no_opts.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
Couldn't you add a flag to exampleTool1.py and reflect it in the descriptor instead of duplicating it?